x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/plugin/TextFormat.java

```diff
@@ -32,7 +34,7 @@
 /**
  * Templating class for displaying SQL responses in text formats.
  */
-enum TextFormat {
+enum TextFormat implements MediaType {
```

Comment:
Is the extension necessary?

Reply:
in order to use a MediaType parser for both `TextFormat` and `XContentType` i had to introduce an interface that is implemented by both

Reply:
Right. Thanks.